avm/res/network/application-gateway/main.bicep

```diff
@@ -79,15 +81,11 @@ param rewriteRuleSets array = []
 
 @description('Optional. The name of the SKU for the Application Gateway.')
 @allowed([
-  'Standard_Small'
-  'Standard_Medium'
-  'Standard_Large'
-  'WAF_Medium'
-  'WAF_Large'
+  'Basic'
   'Standard_v2'
   'WAF_v2'
 ])
-param sku string = 'Standard_v2'
+param sku string = 'WAF_v2'
```

Comment:
I understand this change is based on well-architected framework. However, I am concerned about this breaking deployments for users who have relied on the default value of `Standard_v2` so far. 

@krbar not sure how we can handle such a change but maybe @matebarabas or @alex-frankel can chime in.

Reply:
Hey @ilhaan,

Understand your concern, but this is the AVM specification and standards we must comply with.

To handle the breaking change element here, we will bump the minor version e.g. from `0.1.0` to `0.2.0` and as consumers have to pin to a specific module version this will prevent them from automatically inheriting the breaking change.

We are also brining a changelog capability for each modules README that will help with documentation for these kinds of scenarios that you can see the docs are published ready for here: https://azure.github.io/Azure-Verified-Modules/spec/BCPNRF22/ - tagging @ReneHezser who is driving this feature addition for any ETAs 👍